pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module.adoc

```diff
@@ -0,0 +1,20 @@
+include::../attributes.adoc[]
+[id='module_{context}']
+= Module
```

Comment:
In my opinion, this title doesn't really tell us what the concept being explained is.